R/model_rent_assistance.R

```diff
@@ -0,0 +1,32 @@
+#' Model Rent Assistance
+#' 
+#' @export 
+#' 
+#' @examples
+#' sample<- CJ(rent = 1:500, n_dependants = 0:3, has_partner = 0:1 > 0, is_homeowner = 0:1 > 0, lives_in_sharehouse = 0:1 > 0) 
+#' model_rent_assistance(sample, .Prop_rent_paid_by_RA = 0.75, Max_rate = 500, Min_rent = 100)
```

Comment:
Running this example gives some very strange values like 1.087767e-315 for the final element. I'm pretty sure this is occuring in the rent_assistance function as if you run  `rent_assistance(fortnightly_rent = 1:500, .prop_rent_paid_by_RA = 0.75, max_rate = 500, min_rent = 100)` you get similarly strange outputs

Reply:
Good find, `pminV` expects equal input so we get undefined behaviour: 

```r
pminV(1:6, 1:5)
```